docs/topics/api/github.rst

```diff
@@ -0,0 +1,29 @@
+===============
+GitHub Webhooks
+===============
+
+.. note::
+    
+    This is an Experimental API. We aren't sure of the value of this API yet, so we'd like to see how widely it's used before committing to long term support.
```

Comment:
"This is an experimental API and can change at any time." or something like that maybe? I like to point out to people that things are expected to be changed, also, explicit is better than implicit.